projects/sandbox/timeslide_injections/timeslide_injections/utils.py

```diff
@@ -209,3 +209,20 @@ def check_segment(
                 break
 
     return segment_shifts
+
+
+def chunk_segments(segments: List[tuple], chunk_size: float):
+    out_segments = []
+    for segment in segments:
+        start, stop = segment
+        duration = stop - start
+        if duration > chunk_size:
+            num_segments = int((duration - 1) // chunk_size) + 1
+            logging.info(f"Chunking segment into {num_segments} parts")

```

Comment:
Might suggest that this could be a debug, and might help to clarify which segment is being broken up, but neither is major